Pol 351 azure schedule instance

gopinathmahakud1 opened the pull request:
### Description

-Implemented new policy for azure schedule intance

### Issues Resolved

- New .pt, readme and changelog created.
- Policy link added in the root readme file.

### Contribution Check List

- [x] All tests pass.
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

https://governance.rightscale.com/org/1105/projects/60073/policy-incidents/5f3aaf40a1dbf80001e012a7
![image](https://user-images.githubusercontent.com/55230965/90420093-21ec1f80-e0d5-11ea-8111-b713d928c16e.png)

cdwilhelm commented on cost/azure/schedule_instance/README.md:
```diff
@@ -0,0 +1,74 @@
+
+## What it does
+
+This Policy Template allows you to schedule start and stop times for your instance, along with the option to terminate instance, update and delete schedule.         
```
Please include the cloud name in What it does

cdwilhelm commented on cost/azure/schedule_instance/README.md:
```diff
@@ -0,0 +1,74 @@
+
+Required permissions in the provider:
+
+- Microsoft.Compute/virtualMachines/read
```
Is there a permission to modify the tags?

gopinathmahakud1 replied:
Microsoft.Compute/virtualMachines/write --Creates a new virtual machine or updates an existing virtual machine

To add and update tags we are calling update VM API.

cdwilhelm commented on cost/azure/schedule_instance/azure_schedule_instance.pt:
```diff
@@ -0,0 +1,537 @@
+info(
+  version: "2.0",
+  provider: "Azure",
+  service: "azure",
```
I believe service should be compute.

cdwilhelm commented on cost/azure/schedule_instance/azure_schedule_instance.pt:
```diff
@@ -0,0 +1,537 @@
+      field "subscriptionId" do
+        label "SubscriptionId"
+      end
+      field "subscriptionName" do
```
subscription name should be the first column and sorted.

gopinathmahakud1 replied:
Do we need to sort the subscription, since now we can sort in UI ?

cdwilhelm replied:
Why wouldn't we want to have the subscription sorted in the default view?

gopinathmahakud1 replied:
Yes we can do that, I will update the template.

cdwilhelm commented on cost/azure/schedule_instance/azure_schedule_instance.pt:
```diff
@@ -0,0 +1,537 @@
+      field "rg" do
+        label "Resource Group"
+      end
+      field "location" do
```
the second column should probably be location,

cdwilhelm commented on cost/azure/schedule_instance/azure_schedule_instance.pt:
```diff
@@ -0,0 +1,537 @@
+    escalate $esc_delete_schedule
+    check eq(size(data),0)
+    export do
+      resource_level true
```
Please review the columns and think about how a user may want to view the details as a default view.

gopinathmahakud1 commented:
@cdwilhelm , I have addressed the comments please review the changes.
![image](https://user-images.githubusercontent.com/55230965/90527741-82d82e00-e18f-11ea-8e43-90e3098611e3.png)
